src/components/Home/HomePage.jsx

```diff
@@ -0,0 +1,302 @@
+									className="margin-0 cart-img"
+									src="../images/pc-svg-1.svg"
+									alt=""
+								/>
+							</section>
+
+							<span className="card-text-container card-brand-text">
+								<span className="card-description">
+									<h5>
+										Product description One
+									</h5>
+									<span className="rating-container"></span>
+									<h6>product type</h6>
+								</span>
+
+								<span className="card-container-price-icons card-footer">
+									<span className="card-price">
+										<span>$200</span>
+										<del>$300</del>
+									</span>
+									<span className="optionContainer">
+										<button className="card-wishlist-icons buttonHoverShadow icons-btn-hover AvatarImage flex-row-center primary-btn padding-normal">
+											Add
+											<i className="material-icons">
+												add_shopping_cart
+											</i>
+										</button>
+									</span>
+								</span>
+							</span>
+						</div>
+
+						<div className="mid-card-container sm-card-shadow padding-normal flex-row-center AlertSuccess imgTransition">
+							<section className="flex-row-start">
+								<img
+									className="margin-0 cart-img"
+									src="../images/pc-svg-1.svg"
+									alt=""
+								/>
+							</section>
+
+							<span className="card-text-container card-brand-text">
+								<span className="card-description">
+									<h5>
+										Product description One
+									</h5>
+									<span className="rating-container"></span>
+									<h6>product type</h6>
+								</span>
+
+								<span className="card-container-price-icons card-footer">
+									<span className="card-price">
+										<span>$200</span>
+										<del>$300</del>
+									</span>
+									<span className="optionContainer">
+										<button className="card-wishlist-icons buttonHoverShadow icons-btn-hover AvatarImage flex-row-center primary-btn padding-normal">
+											Add
+											<i className="material-icons">
+												add_shopping_cart
+											</i>
+										</button>
+									</span>
+								</span>
+							</span>
+						</div>
+					</div>
+				</section>
+
+				{/* <!-------------------------------BestSelling Products End-------------------------------> */}
+
+				<div className="mid-container AlertError col-12">
+					{/* <!-------------------------Card Container start--------------------------> */}
+					<div className="card-container buttonHoverShadow">
+						<span
+							className="card-container card-box cardBadge box-shadow"
+							data-label="Trending"
+						>
+							<section className="img-card-offer padding-normal">
+								<img
+									className="imgcard"
+									src="../images/pc-svg-1.svg"
+									alt=""
+								/>
+								<button className="material-icons-text card-wishlist-icons buttonHoverShadow AvatarImage AvatarIcons flex-row-center icon-wishlist">
+									<i className="material-icons">
+										favorite
+									</i>
+								</button>
+							</section>
+
```

Comment:
You can make card component. And use that by using map function

Reply:
Okay, I'll do that. Thank's Vishal for PR review